recipes/README.md

```diff
@@ -0,0 +1,37 @@
+# Run SonarQube with a PostgreSQL database
+
+## Requirements
+
+ * Docker Engine 1.9
+ * Docker Compose 1.6
+
+## Compose file
+
+Download `docker-compose.yml` file and use [docker-compose](https://github.com/docker/compose) to start the containers.
```

Comment:
I suppose that in this case `docker-compose.yml` can become link to an actual file.